lms/product/plugin/plugin.py

```diff
@@ -31,9 +31,9 @@ def __get__(self, instance, owner):
             setattr(instance, self.plugin_name, plugin)  # Overwrite the attr
             return plugin
 
-    grouping: GroupingPlugin = _LazyPlugin()
-    course_copy: CourseCopyPlugin = _LazyPlugin()
-    misc: MiscPlugin = _LazyPlugin()
+    grouping: GroupingPlugin = _LazyPlugin()  # type: ignore
```

Comment:
This code it's a bit magical. Not worth trying to fix this.

Reply:
Should the comment be `# type:ignore`?

```suggestion
    grouping: GroupingPlugin = _LazyPlugin()  # type:ignore
```

Reply:
Went for the type casting :+1: